src/bb-library/Box/Log.php

```diff
@@ -41,8 +41,13 @@ class Box_Log
          self::INFO     => 'INFO',
          self::DEBUG    => 'DEBUG',
     );
+<<<<<<< HEAD
```

Comment:
Conflict resolution required.
@timothygwebb kindly revert this file to the original state. 
I have found what is causing the "Trying to access array offset on value of type null"  error and the solution is not in this particular file. 
I will submit a PR to fix that.

Reply:
@samapraku  Will do and thanks.